addons/spreadsheet/static/src/chart/odoo_chart/odoo_bar_chart.js

```diff
@@ -86,6 +86,16 @@ function getBarConfiguration(chart, labels, locale) {
             ticks: {
                 color,
                 // y axis configuration
+                callback: (value) => {
+                    value = Number(value);
```

Comment:
Don't think you have to check this, formatValue should handle both string an number values already

Reply:
You're right, the `Number` conversion is redundant here since `formatValue` already handles both strings and numbers. I've cleaned this up :)